docpages/disdppgloss.md

```diff
@@ -0,0 +1,51 @@
+
+#### Glossary
+
+Listed in alphabetical order, with terms in bold, here are the basics on...
+
+1. **Action row**: A collection of up to five **components**, which is attached to a message
+
+2. **Audit log**: A log of **events**, that have happened in a **guild**
+
+3. **Auto mod**: Discord's low-code solution to moderation, however it is very limited in scope
+
+4. **Badge**: A decoration on someone's profile showing certain things about them, such as if they have nitro, if they are a discord developer, etc.
+
+5. **Bot token**: A secret string of characters, this is how you login to your bot, if you lose it or it gets leaked, you will have to get a new one from the discord developer portal, and it is necessary to use in your bot
+
+6. **Button**: A **component** on a message that can be styled, that sends an **event** when clicked on by a user
+
+7. **Cache**: A type of storage efficient for things like messages
+
+8. **Callback**: While not strictly related to discord, it is used a LOT in D++, so a callback is when a function is passed to another function, sort of like how you might give someone a telephone number (you give them the means to do some sort of interaction rather than asking them how to interact), which is used to handle responses to **events**
+
+9. **Cluster**: A singular bot application, which is composed of one or more **shards**, a **cluster** is the center of bot development 
+
+10. **(Slash) command**: The primary way a user interacts with a bot, it is a command sent to the bot with **parameters**, which may be optional, and is initiated by staring a message with `/`.
+
+11. **Component**: A component is anything that can appear in a bot's message besides text, such as **buttons** and **select menus**.
+
+12. **Drop down/Select menu**: A **component** of a message that upon being clicked, drops down and allows the user to select an option
+
+13. **Embeds**: A widget attached to a message, which can contain multiple fields of texts, an image, and much more information 
+
+14. **Ephemeral**: A direct message to a user in a public channel, eventually disappears, it is mainly used to give information to a user 
+
+15. **Event**: Something that a Discord bot can respond to, such as a message being sent, a **button** being clicked, or an option being selected, among others.
+
+16. **Guild**: What the Discord API (and most libraries for it) call a server
+
+17. **Intents**: Similar to a guild, a name for permissions of a bot
+
+18. **Interaction**: Something that can be responded to, and it is the main part of an **event** that will be accessed in an application 
+
+19. **Modal**: A pop up form that contains text that can be sent by a bot
+
+20. [**Shards and clusters**](https://dpp.dev/clusters-shards-guilds.html): A cluster is the main container for an application, which contains many shards, each of which manage a subset of your workload, fortunately, D++ does this automatically 

```

Comment:
Can you please replace the url with `\ref clusters-shards-guilds` like @Andosius already said 😀